cmd/alerts/list_silence.go

```diff
@@ -0,0 +1,97 @@
+package alerts
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+func NewCmdListSilence() *cobra.Command {
+	return &cobra.Command{
+		Use:               "list-silence <cluster-id>",
+		Short:             "list all silence",
+		Long:              `list all  silence`,
+		Args:              cobra.ExactArgs(1),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			ListSilence(args[0])
+		},
+	}
+}
+
+//osdctl alerts list-silence ${CLUSTERID} 
+func ListSilence(clusterID string){
+	_, kubeconfig, clientset, err := GetKubeCli(clusterID)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	output, err := getListSilence(kubeconfig, clientset, LocalHostUrl, PodName)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Println("Print information of all silences", output)
+}
+
+//oc -n openshift-monitoring exec alertmanager-main-0 -- amtool silence \
+//--alertmanager.url="http://localhost:9093"
+
+func getListSilence(kubeconfig *rest.Config, clientset *kubernetes.Clientset, LocalHostUrl string, PodName string) (string, error) {
```

Comment:
`getListSilence` feels a bit double to me.
Maybe just `getSilence` or `listSilence` - probably `getSilence`, because the function itself doesn't yet print them which would sound more like `listSilence`.

Reply:
@bergmannf 

Please check https://github.com/openshift/osdctl/pull/500/commits/2c49c094ad8077e3bf0bce597ba16f3dab524470#diff-668108f54ef90b1474162572eff29e8f76603f767259919dfc9e1138e24c5846